ReferenceError: waitDuration is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/workflows/remind_workflow.js:11:30)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Workflow.js:59:59)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
ReferenceError: waitDuration is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/workflows/remind_workflow.js:11:30)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Workflow.js:59:59)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
ReferenceError: waitDuration is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/workflows/remind_workflow.js:11:30)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Workflow.js:59:59)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
ReferenceError: waitDuration is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/workflows/remind_workflow.js:11:30)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Workflow.js:59:59)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
ReferenceError: waitDuration is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/workflows/remind_workflow.js:11:30)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Workflow.js:59:59)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
ReferenceError: reminderDetails is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/workflows/remind_workflow.js:11:30)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Workflow.js:59:59)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
ReferenceError: chatkit is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/actions/send_reminder.js:12:9)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Worker.js:41:41)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
TypeError: this.chatkit.sendSimpleMessage is not a function
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/actions/send_reminder.js:12:22)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Worker.js:41:41)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
TypeError: this.chatkit.sendSimpleMessage is not a function
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/actions/send_reminder.js:12:22)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Worker.js:41:41)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
TypeError: this.chatkit.sendSimpleMessage is not a function
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/actions/send_reminder.js:12:22)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Worker.js:41:41)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
ReferenceError: reminderDetails is not defined
    at Object.handle (/Users/zan/dev/chatbot-reminderer/server/workflows/remind_workflow.js:11:39)
    at _callee$ (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/lib/Worker/async/Workflow.js:59:59)
    at tryCatch (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:288:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
    at _next (/Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at /Users/zan/.zenaton/lib/worker-0.6.0/priv/javascript/node_modules/@babel/runtime/helpers/asyncToGenerator.js:21:12
